receiver/k8sobjectsreceiver/config_test.go

```diff
@@ -135,7 +135,7 @@ func TestValidateResourceConflict(t *testing.T) {
 func TestInvalidPullConfig(t *testing.T) {
```

Comment:
Should this test be renamed? And is there still value in having a test for an invalid pull config on top of a valid pull config?

Reply:
@codeboten yeah, fixed.
I changed the test name, thnks